### DISCUSSION TITLE
Please Deploy Code On Github Pages

### DISCUSSION THREAD

**Discussion by saksham-1304** [recodehive/awesome-github-profiles#1126] [General]
@sanjay-kv Please deploy code on the github pages so that all the changes made so far visible
fix #1095 
![image](https://github.com/user-attachments/assets/27b1b773-3ec7-4317-b059-a6dea7b1ff9d)

**Comment by sanjay-kv** [comment]
Its already on github pages.

**Reply by saksham-1304** [reply]
@sanjay-kv I mean re-deploy ,it was deployed 2 weeks back 
People are raising issue for the bugs that are already fixed .
